docs/dev/plugins.rst

```diff
@@ -211,6 +211,22 @@ The events currently available are:
 * *import_begin*: called just before a ``beet import`` session starts up.
   Parameter: ``session``.
 
+* *trackinfo_received*: called after meta data for a track item has been fetched
+  from disparate sources, such as MusicBrainz. Gives a developer the option to
+  intercept the fetched TrackInfo object. Can be used to modify tags on a ``beet
+  import`` operation or during later adjustments, such as ``mbsync``. Can be
+  slow, as event is fired for any fetched possible match *before* user or
```

Comment:
I actually meant that the event is fired for all received `TrackInfo` and `AlbumInfo` objects, regardless of the user's later selection. Slow handlers consequently do slow down the flow, which results in the user having to wait longer before he can make his selection.